sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRegistryTestEnvironment.cs

```diff
@@ -10,5 +10,8 @@ public class ContainerRegistryTestEnvironment : TestEnvironment
         public string Endpoint => GetRecordedVariable("CONTAINERREGISTRY_ENDPOINT");
         public string UserName => GetRecordedVariable("CONTAINERREGISTRY_USERNAME", options => options.IsSecret());
         public string Password => GetRecordedVariable("CONTAINERREGISTRY_PASSWORD", options => options.IsSecret());
+        public string Registry => GetRecordedVariable("CONTAINERREGISTRY_REGISTRY_NAME");
+
+        public bool IsTestModeLive => this.Mode != RecordedTestMode.Playback;
```

Comment:
nit: this property is a little confusing since it will be true for both Live and Recorded mode.
Suggestion: just check for mode != playback directly.

Reply:
I copied this from @pakrym's monitor code ... @pakrym, was there thinking behind naming this property this way?

Reply:
The way you are using it you don't need it at all. RecodededTestBase has a Mode property of its own.